src/view/frontend/templates/translation.phtml

```diff
@@ -0,0 +1,8 @@
+<?php
+use Magento\Framework\Escaper;
+
+/** @var \Hyva\Checkout\Block\CheckoutTranslator $block  */
+?>
+<script>
+    var CHECKOUT_TRANSLATIONS = '<?= $block->getTranslations()  ?>';
```

Comment:
I like that better than a huge `data` attribute (although it's a global object) :+1:

Reply:
Yes, exactly same thought I had